geode-redis/src/main/java/org/apache/geode/redis/internal/statistics/GeodeRedisStats.java

```diff
@@ -72,7 +72,7 @@ public GeodeRedisStats(StatisticsFactory factory, String textId,
 
     type = statisticsTypeFactory
         .createType("RedisStats",
-            "Statistics for a GemFire Redis Server",
+            "Statistics for a Geode Compatible with Redis Server",
```

Comment:
"Statistics for Geode APIs compatible with Redis"

Reply:
I think that "Statistics for a Geode server compatible with Redis" makes more sense. is that ok?

Reply:
Yup!  Thank you!!